bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/form/Field.java

```diff
@@ -15,28 +15,18 @@
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 package com.adobe.cq.wcm.core.components.models.form;
 
-import org.jetbrains.annotations.NotNull;
 import org.osgi.annotation.versioning.ConsumerType;
 
 import com.adobe.cq.export.json.ComponentExporter;
+import com.adobe.cq.wcm.core.components.models.CoreComponent;
 
 /**
  * A base interface to be extended by all the different types of form fields.
  *
  * @since com.adobe.cq.wcm.core.components.models.form 13.0.0
  */
 @ConsumerType
-public interface Field extends ComponentExporter {
-
-    /**
-     * Returns an unique identifier for this field.
-     *
-     * @return an unique identifier for the field
-     * @since com.adobe.cq.wcm.core.components.models.form 13.0.0; marked <code>default</code> in 14.1.0
-     */
-    default String getId() {
-        throw new UnsupportedOperationException();
-    }
+public interface Field extends CoreComponent, ComponentExporter {
```

Comment:
@gitrajat Corecomponent is already extending componentExplorer. Do we need to again specify ComponentExporter here?